docs/cli/dotnet-new.md

```diff
@@ -0,0 +1,39 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new sample .NET Core project
+
+## SYNOPSIS
+dotnet new
```

Comment:
nearly all of the other synopsis are 'dotnet-verb'. why isn't this 'dotnet-new'?